prometheus/testutil/promlint/promlint.go

```diff
@@ -313,9 +313,10 @@ var (
 		// Base units.
 		"amperes": "amperes",
 		"bytes":   "bytes",
-		"celsius": "celsius", // Celsius is more common in practice than Kelvin.
+		"celsius": "celsius", // Celsius is preferred in computing.
```

Comment:
How about "Also allow Celsius because it is common in typical Prometheus use cases."